compositor_pipeline/src/pipeline/whip_whep/config.rs

```diff
@@ -0,0 +1,53 @@
+use std::{env, str::FromStr};
+
+#[derive(Debug, Clone)]
+pub struct Config {
+    pub api_port: u16,
+    pub start_whip_whep: bool,
+}
+
+#[derive(Debug, Copy, Clone)]
+pub enum LoggerFormat {
+    Pretty,
+    Json,
+    Compact,
+}
+
+impl FromStr for LoggerFormat {
```

Comment:
don't read envs from inside this crate, env should only be read in top level crate and passed down via options